sdk/elixir/mix.exs

```diff
@@ -0,0 +1,54 @@
+defmodule Dagger.MixProject do
+  use Mix.Project
+
+  @version "0.2.0-dev"
+  @source_url "https://github.com/wingyplus/dagger_ex"
+
+  def project do
+    [
+      app: :dagger_ex,
+      version: @version,
+      elixir: "~> 1.14",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      package: package(),
+      docs: docs()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  defp deps do
+    [
+      {:req, "~> 0.3"},
+      {:absinthe_client, "~> 0.1"},
+      {:nimble_options, "~> 1.0"},
+      {:ex_doc, "~> 0.27", only: :dev, runtime: false}
+    ]
+  end
+
+  defp package do
+    %{
+      name: "dagger_ex",
+      description: "Dagger SDK for Elixir",
+      licenses: ["MIT"],
+      links: %{
+        "GitHub" => "https://github.com/dagger/dagger",
+        "Changelog" => "#{@source_url}/blob/sdk/elixir/v#{@version}/CHANGELOG.md"
```

Comment:
This is just a link that shows on https://hex.pm/packages/dagger_ex. We can use a release note from GitHub release page instead.